package.json

```diff
@@ -78,6 +78,7 @@
   },
   "devDependencies": {
     "@blueprintjs/core": "^2.1.1",
+    "@blueprintjs/docs-theme": "^3.2.0",
```

Comment:
sorry im a little confused! why are we adding a documentation dep in this pr?

Reply:
Yea it is redundant, we have removed it.